source/JABHandler.py

```diff
@@ -1,4 +1,4 @@
-#javaAccessBridgeHandler.py
+﻿#javaAccessBridgeHandler.py

```

Comment:
nit: A Unicode BOM has been added here. Frustratingly, this breaks xgettext. Please remove it. I'll update the Contributing document to mention this.